### PR TITLE
Package file split current dependencies into dependencies and devdepe…

### PR DESCRIPTION
**Pull request by ShahAaron**
…ndencies

## Summary

Addresses issue #1032 

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -129,26 +111,65 @@
     "@wordpress/url": "^2.9.0",
     "accessible-autocomplete": "^1.6.2",
     "autoprefixer": "^9.5.0",
+    "classnames": "^2.2.6",
+    "cssnano": "^4.1.10",
```
Comment (aaemnnosttv) [inline package.json:115]: `cssnano` is a build dependency.

File: package.json [hunk 2]
```diff
@@ -129,26 +111,65 @@
...
+    "dompurify": "^2.0.8",
+    "element-closest": "^2.0.2",
+    "focus-trap-react": "^5.0.1",
+    "formdata-polyfill": "^3.0.17",
+    "intl": "^1.2.5",
+    "intl-locales-supported": "^1.6.0",
+    "invariant": "^2.2.4",
+    "jquery": "^3.4.1",
```
Comment (aaemnnosttv) [inline package.json:123]: @felixarntz @tofumatt do you know why we have `jquery` in here?

File: package.json [hunk 3]
```diff
@@ -129,26 +111,65 @@
...
+    "lodash": "^4.17.15",
+    "md5": "^2.2.1",
+    "natives": "^1.1.6",
+    "polyfill-library": "^3.31.1",
+    "promise-polyfill": "^8.0.0",
+    "punycode": "^2.1.1",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "require-dir": "^1.2.0",
```
Comment (aaemnnosttv) [inline package.json:132]: Build dependency 😄 
https://github.com/google/site-kit-wp/blob/46290f84ed6897f18c094e57949fcddeac7e7587/gulpfile.babel.js#L23

File: package.json [hunk 4]
```diff
@@ -129,26 +111,65 @@
...
+    "sanitize-filename": "^1.6.3",
```
Comment (aaemnnosttv) [inline package.json:133]: Build dependency
https://github.com/google/site-kit-wp/blob/46290f84ed6897f18c094e57949fcddeac7e7587/gulp-tasks/zip.js#L10

File: package.json [hunk 5]
```diff
@@ -129,26 +111,65 @@
...
+    "svgo": "^1.3.0",
```
Comment (aaemnnosttv) [inline package.json:134]: Pretty sure this is used by gulp

File: package.json [hunk 6]
```diff
@@ -129,26 +111,65 @@
...
+    "svgxuse": "^1.2.6",
```
Comment (aaemnnosttv) [inline package.json:135]: I'm not sure about this one... @tofumatt ?

File: package.json [hunk 7]
```diff
@@ -129,26 +111,65 @@
...
+    "@babel/preset-react": "^7.0.0",
+    "@babel/register": "^7.4.4",
+    "@babel/traverse": "^7.5.5",
+    "@storybook/addon-viewport": "^5.0.11",
+    "@storybook/react": "^5.0.11",
+    "@testing-library/jest-dom": "^4.1.2",
+    "@testing-library/react": "^9.4.1",
+    "@wordpress/babel-preset-default": "^4.5.0",
+    "@wordpress/browserslist-config": "^2.6.0",
+    "@wordpress/dom-ready": "^2.5.1",
```
Comment (aaemnnosttv) [inline package.json:156]: Client side dependency
https://github.com/google/site-kit-wp/blob/46290f84ed6897f18c094e57949fcddeac7e7587/assets/js/googlesitekit-activation.js#L24

File: package.json [hunk 8]
```diff
@@ -160,32 +181,18 @@
...
-    "lodash": "^4.17.15",
-    "md5": "^2.2.1",
     "mini-css-extract-plugin": "^0.4.5",
-    "natives": "^1.1.6",
     "node-sass": "^4.11.0",
-    "polyfill-library": "^3.31.1",
     "postcss-loader": "^3.0.0",
     "postcss-preset-env": "^6.6.0",
-    "promise-polyfill": "^8.0.0",
     "prop-types": "^15.7.2",
```
Comment (aaemnnosttv) [inline package.json:191]: `prop-types` is a normal dependency

File: package.json [hunk 9]
```diff
@@ -160,32 +181,18 @@
...
     "psl": "^1.4.0",
```
Comment (aaemnnosttv) [inline package.json:192]: `psl` is a normal dependency

**Comment by ShahAaron** [comment]
@googlebot I consent

**Comment by aaemnnosttv** [comment]
@ShahAaron looks like you've included a bunch of other commits here from another branch perhaps? Can you take a look and see what happened there? That's why you're seeing the CLA notice here where you shouldn't.

**Pull request closed without merging**